### DISCUSSION TITLE
Combining multiple "commands" to create one QR code

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#462] [Q&A] [answered]
Sometimes, I would like to switch off the NR01, and reset BITR, WIDE and LOGB, so I can just use the "natural" colour setting to avoid colour grading. Then I have to scan them one by one to get each setting back.

Is there any way I can combine the "commands" that I want with their respective values and generate them using any QR code generator and produce only one QR code to scan and GoPro restores all the commands that I have encoded?

Thanks.

**Comment by dnewman-gpsw** [comment, marked as answer]
Just pack them all together. No commas, no spaces.  oMNR01=1oMBITR=170 etc.

**Reply by tetrahydra** [reply]
To make it permanent, replace oM with!M:

!MNR01=1!MBITR=170
